teams/vsp/teams/insights-analytics/ga-events-data-dictionaries.md

```diff
@@ -742,116 +754,196 @@ _When the orientation of the way search results are presented is changed, either
+| Product - Disability Ratings Viewed | 60 | User | Value is set to true when a user views disability ratings |
+| Video Title | 61 | Hit | Title of a video (for example, there is a video titled `How to Help Homeless Veterans on this page: www.va.gov/homeless/) |
+| Product - Health Care VAOS User | 62 | User | A value set to true if a user is logged in as a health care VAOS user |
+| Product Area Used - Disability | 63 | User | Distinguish users who have ever completed at least one transaction in the Disability product area |
+| Education - Benefit Update | 64 | Hit | User updates their benefits within the education benefit hub (values for this CD include (1) new (2) update (3) stem-scholarship) |
+| Education - Benefit Claim for Self | 65 | Hit | User's response to the question of if they are filling out the benefit claim for themselves. Options include: (1) yes (2) no (3) unanswered  |
+| Education - National Call to Service | 66 | Hit | User's response to the question of national call to service. Options include: (1) yes (2) no (3) unanswered. Found on this page: www.va.gov/education/eligibility/ as well as this page: www.va.gov/education/how-to-apply/ |
+| Education - Applicant is Vet Tec | 67 | Hit | User's response to the question of vet tec. Options include: (1) yes (2) no (3) unanswered. Found on this page: www.va.gov/education/eligibility/ as well as this page: www.va.gov/education/how-to-apply/ |
+| Education - Sponsor Transferred Benefits | 68 | Hit | User's response to the question of sponsor transfer benefits. Options include: (1) yes (2) no (3) unanswered. Found on this page: www.va.gov/education/eligibility/ as well as this page: www.va.gov/education/how-to-apply/ |
+| Education - Receiving Sponsor Benefits | 69 | Hit | User's response to the question of receiving sponsor benefits. Options include: (1) yes (2) no (3) unanswered. Found on this page: www.va.gov/education/eligibility/ as well as this page: www.va.gov/education/how-to-apply/ |
+| Education - Sponsor Available | 70 | Hit | User's response to the question of sponsor available. Options include: (1) yes (2) no (3) unanswered. Found on this page: www.va.gov/education/eligibility/ as well as this page: www.va.gov/education/how-to-apply/ |
+| Health Care VAOS - Appointment Type | 71 | Hit | Appointment type (either `confirmed` or `pending`) for a VAOS apointment |
+| Health Care VAOS - Type of Care | 72 | Hit | Answer to the question asking users to `choose the type of care you need`. Options include: `primary care`, `mental health`, `optometry`, `COVID-19 vaccine`, etc. Pages this CD is set: www.va.gov/health-care/schedule-view-va-appointments/appointments/new-appointment |
+| Health Care VAOS - Flow | 73 | Hit | Type of flow for a VAOS request. Options include: `va-request`, `direct`, and `cc-request` |
+| Education - 22-1995 - Exhausted All Benefits | 74 | Hit | **No data is being collected** |
```

Comment:
If no data is being collected here let's open a bug and put in the backlog, low priority